src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx

```diff
@@ -1,69 +1,49 @@
 import {
   Footnote,
-  FootnoteMeta,
   FootnoteProps,
-  FootnoteMetaGetters,
 } from '@admin/services/release/edit-release/footnotes/types';
 import footnotesService from '@admin/services/release/edit-release/footnotes/service';
-import permissionService from '@admin/services/permissions/service';
-import { generateFootnoteMetaMap } from '@admin/services/release/edit-release/footnotes/util';
 import Link from '@admin/components/Link';
 import withErrorControl, {
   ErrorControlProps,
 } from '@admin/validation/withErrorControl';
 import LoadingSpinner from '@common/components/LoadingSpinner';
 import ModalConfirm from '@common/components/ModalConfirm';
-import React, { useCallback, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import FootnotesList from '@admin/components/footnotes/FootnotesList';
 import FootnoteForm, {
   FootnoteFormConfig,
 } from '@admin/components/footnotes/form/FootnoteForm';
+import { FootnotesData } from '@admin/pages/release/edit-release/data/ReleaseDataPage';
 
 interface Props {
   publicationId: string;
   releaseId: string;
+  footnotesPropData?: FootnotesData;
+  onSubmit: (footnotesData: FootnotesData) => void;
+  getFootnoteData: () => void;
 }
 
 const ReleaseFootnotesSection = ({
-  publicationId,
-  releaseId,
+  footnotesPropData: footnotesData,
+  onSubmit,
+  getFootnoteData,
   handleApiErrors,
 }: Props & ErrorControlProps) => {
   const [loading, setLoading] = useState<boolean>(true);
-  const [footnoteMeta, setFootnoteMeta] = useState<FootnoteMeta>();
-  const [footnotes, setFootnotes] = useState<Footnote[]>([]);
   const [footnoteForm, _setFootnoteForm] = useState<FootnoteFormConfig>({
     state: 'cancel',
   });
-  const [footnoteMetaGetters, setFootnoteMetaGetters] = useState<
-    FootnoteMetaGetters
-  >();
+
   const [footnoteToBeDeleted, setFootnoteToBeDeleted] = useState<
     Footnote | undefined
   >();
-  const [hasSufficientData, setHasSufficientData] = useState<boolean>(true);
-  const [canUpdateRelease, setCanUpdateRelease] = useState(false);
-
-  const getFootnoteData = useCallback(() => {
-    setLoading(true);
-    Promise.all([
-      footnotesService.getReleaseFootnoteData(releaseId),
-      permissionService.canUpdateRelease(releaseId),
-    ])
-      .then(([{ meta, footnotes: footnotesList }, canUpdateReleaseResult]) => {
-        setFootnoteMeta(meta);
-        setHasSufficientData(!!Object.keys(meta).length);
-        setFootnotes(footnotesList);
-        setFootnoteMetaGetters(generateFootnoteMetaMap(meta));
-        setLoading(false);
-        setCanUpdateRelease(canUpdateReleaseResult);
-      })
-      .catch(handleApiErrors);
-  }, [handleApiErrors, releaseId]);
 
   useEffect(() => {
-    getFootnoteData();
-  }, [publicationId, releaseId, handleApiErrors, getFootnoteData]);
+    if (footnotesData) {
+      setLoading(footnotesData.loading);
```

Comment:
As I've mentioned to you in the other PR, mutating state is a big no-no 💀 

You shouldn't even need this as `loading` should be controlled as a prop that is passed in from the parent component.